test/core_checkM_test.py

```diff
@@ -29,12 +26,80 @@
 from kb_Msuite.Utils.OutputBuilder import OutputBuilder
 
 
+TEST_DATA = {
```

Comment:
Data structure to store the test data, rather than having file names / paths scattered through the code.